### DISCUSSION TITLE
2.3.30 Distributed Deployment Not Updatng Minions

### DISCUSSION THREAD

**Discussion by pappajohndd** [Security-Onion-Solutions/securityonion#3525] [Unsupported Versions] [answered]
Upgraded manager about 28 hours ago. Search, and 2 sensor nodes have not upgraded. Is there a way to turn off updates from the manager and run soup on each node?

**Comment by pappajohndd** [comment]
still not updating

**Comment by TOoSmOotH** [comment, marked as answer]
Did you try connecting to a node and running a highstate manually to see what the problem is?

**Reply by pappajohndd** [reply]
Yes showed no problems. Now I rebooted the nodes and they are all wait_start on all services

**Reply by TOoSmOotH** [reply]
How many states did it run when you did a highstate? On one of the non managers check the bootstrap log in /tmp. My guess is it is unable to upgrade salt

**Reply by pappajohndd** [reply]
[1;32m *  INFO[0m: Running install_centos_stable_deps()
Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=stock error was
12: Timeout on http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=stock: (28, 'Operation too slow. Less than 1000 bytes/sec transferred the last 30 seconds')
Cannot retrieve metalink for repository: epel/x86_64. Please verify its path and try again
warning: /var/tmp/rpm-tmp.WIDy4Z: Header V4 RSA/SHA256 Signature, key ID 352c64e5: NOKEY

Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=stock error was
12: Timeout on http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=stock: (28, 'Operation too slow. Less than 1000 bytes/sec transferred the last 30 seconds')

Cannot find a valid baseurl for repo: base/7/x86_64
[1;31m * ERROR[0m: Failed to run install_centos_stable_deps()!!!

**Reply by TOoSmOotH** [reply]
Assuming you have updates through the manager, try running this on the manager: 
sudo docker stop so-aptcacherng
sudo salt-call state.highstate

**Reply by pappajohndd** [reply]
do I then need to run highstate on nodes?

**Reply by TOoSmOotH** [reply]
on one of them yes.. see if that addresses the issue.

**Reply by pappajohndd** [reply]
did not resolve the issue

**Reply by TOoSmOotH** [reply]
Are you updating through your manager?

**Reply by pappajohndd** [reply]
yes

**Reply by TOoSmOotH** [reply]
sounds like it's having trouble getting to the internet then. What happens when you do a yum update on your manager?

**Reply by pappajohndd** [reply]
yum works on manager

**Reply by pappajohndd** [reply]
but not working so well on nodes

**Reply by TOoSmOotH** [reply]
Do the individual nodes have internet access? If so to get back up and running you could comment out the proxy in /etc/yum.conf

**Reply by pappajohndd** [reply]
so I could run soup from each node?

**Reply by pappajohndd** [reply]
That worked! Thanks for your help! You are my favorite today
